src/App.js

```diff
@@ -0,0 +1,34 @@
+import './styles/App.scss';
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import MainPage from './components/HomePage/MainPage';
+import Details from './components/DoctorsData/DetailsPage/Details';
+import Appointments from './components/AppointmentsData/AppointmentsPage';
+import LoginPage from './components/Authentication/LoginPage';
+import RegistrionPage from './components/Authentication/RegistrionPage';
+import SplashPage from './components/SplashPage/SplashPage';
+import AddItem from './components/DoctorsData/AddItem';
+import DeleteItem from './components/DoctorsData/DeleteItem';
+import AppointmentsDetails from './components/AppointmentsData/DetailsPage/Details';
+import AddAppointmentsForm from './components/AppointmentsData/AddAppointment';
+
+function App() {
```

Comment:
- [ ] The "Add doctor" and "Delete doctor" functionality is not present yet:


![image](https://user-images.githubusercontent.com/57421823/213646890-1c138284-e692-4f5b-a6c1-a3253063c8f9.png)

![image](https://user-images.githubusercontent.com/57421823/213646911-6cdad2fc-0057-4a0e-bd76-2af6b41b9692.png)

One is a core requirement and the other is a requirement for 3-4 members.

I'm pretty confident that you will add that functionality to your next review request.  Please, kindly don't forget to style those sections properly according to the design guidelines. 💪